src/main/components/form/validator.ts

```diff
@@ -30,6 +30,12 @@ export const isContent2000CharsOrLess: Validator = value => {
   }
 };
 
+export const isContent2000CharsOrLess: Validator = value => {
```

Comment:
Might be worth pulling down the latest changes from 1.1.0.7. I see there is similar code in place for it - https://github.com/hmcts/et-sya-frontend/pull/647/files#diff-d2aa0a303ec8b8706c23909545d7cdca4bcf67987651c823fd3071b63d95a5dfR27

Reply:
removed the duplicate code